Library/Homebrew/keg_relocate.rb

```diff
@@ -75,8 +78,8 @@ def text_files
       path.find do |pn|
         next if pn.symlink? || pn.directory?
         next if Metafiles::EXTENSIONS.include? pn.extname
-        if Utils.popen_read("/usr/bin/file", "--brief", pn).include?("text") ||
-           pn.text_executable?
+        if pn.text_executable? ||
+           Utils.popen_read("/usr/bin/file", "--brief", pn).include?("text")
```

Comment:
@reitermarkus Some local testing looks like it should be, good catch. There's probably a maximum number of arguments we can specify but batching them may well produce a speedup 👍